tutorials/cp-sdk-android-flows-passcode/cp-sdk-android-flows-passcode.md

```diff
@@ -0,0 +1,185 @@
+---
+author_name: Haoyue Feng
+author_profile: https://github.com/Fenghaoyue
+title: Handle Passcode with the Flows Component
+description: Learn how to use the Flows component of SAP BTP SDK for Android to handle passcode change, forgetting passcode and passcode verification scenarios.
+auto_validation: true
+time: 60
+tags: [ tutorial>beginner, operating-system>android, topic>mobile, products>sap-business-technology-platform]
+primary_tag: products>sap-btp-sdk-for-android
+---
+
+## Prerequisites
+- You completed [Try Out the SAP BTP SDK Wizard for Android](cp-sdk-android-wizard-app).
+- You completed [Get Familiar with Flows Component by Wizard Generated Application](cp-sdk-android-flows-wizard).
+
+## Details
+### You will learn
+  - How to change passcode with the Flows component
+  - How to handle forgetting passcode scenario with the Flows Component
+  - How to verify passcode with the Flows component
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Handle passcode change with the Flows component)]
+
+[OPTION BEGIN [Java]]
+
+1.  Open the project you previously created using the SAP BTP SDK Wizard for Android.
```

Comment:
```suggestion
1.  Open the project you [previously created](cp-sdk-android-wizard-app) using the SAP BTP SDK Wizard for Android.
```

Reply:
Adding a hyperlink to the required tutorial makes it easier for people to verify if they have created the project

Reply:
Updated for both Java and Kotlin step, thanks.